latest/index.html

```diff
@@ -53,56 +56,107 @@ <h1 id="headline"><a href=""><img src="../images/liquigraph-logo.png" alt="Liqui
-		</div>
-		<div id="spring-boot">
-			<h2>Spring Boot</h2>
-			<p>Save the migration file at <span class='path'>${your_project}/src/main/resources/db/migrations/changelog.xml</span>.</p>
-			<p>Then, include Liquigraph Spring Boot starter in your <span class="path">pom.xml</span>:</p>
-			<pre><code class="xml">&lt;dependency&gt;
+liquigraph.runMigrations(configuration);</code></pre>
+              <p>
+                If want to dry-run your migrations, replace <code>withRunMode()</code> by
+                <code>withDryRunMode(Paths.get(outputDirectory))</code>, where
+                <code>outputDirectory</code> specifies the path of the directory where
+                <code>output.cypher</code> will be written in.
+              </p>
+            </div>
+
+            <div id="spring-boot">
+              <h2>Spring Boot</h2>
+              <ol>
+                <li>
+                  <p>
+                    Include Liquigraph Spring Boot starter in your
+                    <span class="path">pom.xml</span>:
+                  </p>
+                  <pre><code class="xml">&lt;dependency&gt;
     &lt;groupId&gt;org.liquigraph&lt;/groupId&gt;
     &lt;artifactId&gt;liquigraph-spring-boot-starter&lt;/artifactId&gt;
     &lt;version&gt;3.0.2-SNAPSHOT&lt;/version&gt;
 &lt;/dependency&gt;</code></pre>
-			<p>Annotate your main class with <a href="http://docs.spring.io/spring-boot/docs/current/reference/html/using-boot-auto-configuration.html"><code>@org.springframework.boot.autoconfigure.EnableAutoConfiguration</code></a>.</p>
-			<p>Finally, configure <code>liquigraph.url</code>, <code>liquigraph.user</code> and <code>liquigraph.password</code> properties to your <span class="path">application.properties</span> (or another relevant <a href="https://docs.spring.io/spring-boot/docs/current/reference/html/boot-features-external-config.html">configuration source</a>) file and you are good to go!</p>
-			<p>You are now automatically running Liquigraph migrations at startup!</p>
-		</div>
-		<div id="maven">
-			<h2>The Maven way</h2>
-			<p>Save the migration file at <span class='path'>${your_project}/src/main/resources/changelog.xml</span>.</p>
-			<p>Then, include Liquigraph plugin in your <span class="path">pom.xml</span>:
-			<pre><code class="xml">&lt;plugin&gt;
+                </li>
+                <li>
+                  <p>Include a Neo4j JDBC driver in your <span class="path">pom.xml</span>:</p>
+                  <pre><code class="xml">&lt;dependency&gt;
+    &lt;groupId&gt;org.neo4j&lt;/groupId&gt;
+    &lt;artifactId&gt;neo4j-jdbc-bolt&lt;/artifactId&gt;
+    &lt;!--&lt;artifactId&gt;neo4j-jdbc-http&lt;/artifactId&gt;--&gt;
+    &lt;version&gt;3.0.1&lt;/version&gt;
+&lt;/dependency&gt;</code></pre>
+                </li>
+                <li>
+                  <p>Expose a <code>DataSource</code> bean, which is used by Liquigraph.<p>
```

Comment:
I'd provide 1 or 2 options here (like using the JDBC starter or using a 3rd party like HikariCP).